public/js/components/RightSidebar.js

```diff
@@ -48,8 +49,8 @@ function RightSidebar({ resume, command, breakOnNext,
             component: Breakpoints,
             opened: true },
           { header: "Call Stack",
-            component: () => dom.div({className: "pane-info"}, "Not Paused")
-          },
+            component: Frames,
+            componentProps: { frames }},
```

Comment:
I don't love how often we are "connecting" components, and I would prefer to pull state off from above and pass it down more often. This is somewhat subjective, but it's usually preferred to do it sparingly. I'm not clear where the balance is though, and once we implement these last set of features this is a good thing to think about across the project as a whole.